include/amici/defines.h

```diff
@@ -18,37 +18,37 @@ constexpr double pi = 3.14159265358979323846;
 
 // clang-format off
 
-#define AMICI_ONEOUTPUT   5
+constexpr int AMICI_ONEOUTPUT=                 5;
 
 /* Return codes */
-#define AMICI_RECOVERABLE_ERROR          1
-#define AMICI_UNRECOVERABLE_ERROR      -10
-#define AMICI_TOO_MUCH_WORK             -1
-#define AMICI_TOO_MUCH_ACC              -2
-#define AMICI_ERR_FAILURE               -3
-#define AMICI_CONV_FAILURE              -4
-#define AMICI_ILL_INPUT                -22
-#define AMICI_ERROR                    -99
-#define AMICI_NOT_IMPLEMENTED         -999
-#define AMICI_SUCCESS                    0
-#define AMICI_DATA_RETURN                1
-#define AMICI_ROOT_RETURN                2
-
-#define AMICI_NORMAL                     1
-#define AMICI_ONE_STEP                   2
-
-#define AMICI_PREEQUILIBRATE            -1
+constexpr int AMICI_RECOVERABLE_ERROR=         1;
+constexpr int AMICI_UNRECOVERABLE_ERROR=     -10;
+constexpr int AMICI_TOO_MUCH_WORK=            -1;
+constexpr int AMICI_TOO_MUCH_ACC=             -2;
+constexpr int AMICI_ERR_FAILURE=              -3;
+constexpr int AMICI_CONV_FAILURE=             -4;
+constexpr int AMICI_ILL_INPUT=               -22;
+constexpr int AMICI_ERROR=                   -99;
+constexpr int AMICI_NOT_IMPLEMENTED=        -999;
+constexpr int AMICI_SUCCESS=                   0;
+constexpr int AMICI_DATA_RETURN=               1;
+constexpr int AMICI_ROOT_RETURN=               2;
+
+constexpr int AMICI_NORMAL=                    1;
+constexpr int AMICI_ONE_STEP=                  2;
+
+constexpr int AMICI_PREEQUILIBRATE=           -1;
 
 #ifndef booleantype
 #define booleantype int
 #endif
 
 #ifndef FALSE
-#define FALSE 0
+constexpr int FALSE=0;
```

Comment:
With the replacements below, is this still being used anywhere?

Reply:
unfortunately yes, the solver has to use it in a couple of places.

Reply:
done, we are gonna wait for CI for a while anyways :D